### PR TITLE
Add 'es'

### PR DESCRIPTION
**Pull request by gerardojro**
Added 'es' to locales

### CODE REVIEW

**Review by fitzydev — APPROVED** [review]

File: locales/locales.lua [hunk 1]
```diff
@@ -119,5 +119,59 @@ Locales = {
         ['autotheft'] = "Tyveri af et køretøj",
         ['officerdown'] = "KOLLEGA SKADET",
         ['persondown'] = "Personen er såret",
+    },
+    ['es'] = {
+        ['unknown_caller'] = "Llamada Desconocida",
+        ['caller_unknown'] = "Desconocido",
+        ['caller_local'] = "Local",
+        ['call_from'] = "Llamada de ",
```
Comment (MonkeyWhisper) [inline locales/locales.lua:127]: You have space after " Fix that in a lot of stuff

File: locales/locales.lua [hunk 2]
```diff
@@ -119,5 +119,59 @@ Locales = {
...
+        ['two_door'] = "Dos-puertas ",
```
Comment (MonkeyWhisper) [inline locales/locales.lua:128]: Space here after "

File: locales/locales.lua [hunk 3]
```diff
@@ -119,5 +119,59 @@ Locales = {
...
+        ['three_door'] = "Tres-puertas ",
```
Comment (MonkeyWhisper) [inline locales/locales.lua:129]: Space here after "

File: locales/locales.lua [hunk 4]
```diff
@@ -119,5 +119,59 @@ Locales = {
...
+        ['sports'] = "Carro Deportivo",
+        ['super'] = "Super carro",
+        ['motorcycle'] = "Motocicleta",
+        ['offroad'] = "Todo-terreno",
+        ['industrial'] = "Vehiculo Industrial",
+        ['utility'] = "Vehiculo de Utilidad",
+        ['van'] = "Van",
+        ['service'] = "Service vehicle",
+        ['military'] = "Military vehicle",
+        ['truck'] = "Truck",
```
Comment (MonkeyWhisper) [inline locales/locales.lua:146]: Not in spanish

File: locales/locales.lua [hunk 5]
```diff
@@ -119,5 +119,59 @@ Locales = {
...
+        ['sports_classic'] = "Clasico Deportivo",
+        ['sports'] = "Carro Deportivo",
+        ['super'] = "Super carro",
+        ['motorcycle'] = "Motocicleta",
+        ['offroad'] = "Todo-terreno",
+        ['industrial'] = "Vehiculo Industrial",
+        ['utility'] = "Vehiculo de Utilidad",
+        ['van'] = "Van",
+        ['service'] = "Service vehicle",
+        ['military'] = "Military vehicle",
```
Comment (MonkeyWhisper) [inline locales/locales.lua:145]: Not in spanish

File: locales/locales.lua [hunk 6]
```diff
@@ -119,5 +119,59 @@ Locales = {
...
+        ['muscle'] = "Muscle car",
+        ['sports_classic'] = "Clasico Deportivo",
+        ['sports'] = "Carro Deportivo",
+        ['super'] = "Super carro",
+        ['motorcycle'] = "Motocicleta",
+        ['offroad'] = "Todo-terreno",
+        ['industrial'] = "Vehiculo Industrial",
+        ['utility'] = "Vehiculo de Utilidad",
+        ['van'] = "Van",
+        ['service'] = "Service vehicle",
```
Comment (MonkeyWhisper) [inline locales/locales.lua:144]: Not in spanish

File: locales/locales.lua [hunk 7]
```diff
@@ -119,5 +119,59 @@ Locales = {
...
+        ['vangelico'] = "Robo a Joyeria Vangelico",
+        ['houserobbery'] = "Alarma de Casa Activada",
+        ['drugsell'] = "Actividad Sospechosa",
+        ['carjacking'] = "Car Jacking", 
+        ['vehicletheft'] = "Robo de Vehiculo", 
+        ['officerdown'] = "Oficial Herido",
+        ['emsdown'] = "EMS Herido",
+        ['artgalleryrobbery'] = "Robo de Galeria de Arte",
+        ['humanerobbery'] = "Robo de Humane Labs",
+        ['trainrobbery'] = "Robo en el Tren",
```
Comment (MonkeyWhisper) [inline locales/locales.lua:169]: Robo del tren*

File: locales/locales.lua [hunk 8]
```diff
@@ -119,5 +119,59 @@ Locales = {
...
+        ['pacificbank'] = "Robo Banco Pacifico",
+        ['prisonbreak'] = "Fuga de Carcel en Progeso",
+        ['vangelico'] = "Robo a Joyeria Vangelico",
+        ['houserobbery'] = "Alarma de Casa Activada",
+        ['drugsell'] = "Actividad Sospechosa",
+        ['carjacking'] = "Car Jacking", 
+        ['vehicletheft'] = "Robo de Vehiculo", 
+        ['officerdown'] = "Oficial Herido",
+        ['emsdown'] = "EMS Herido",
+        ['artgalleryrobbery'] = "Robo de Galeria de Arte",
```
Comment (MonkeyWhisper) [inline locales/locales.lua:167]: Robo de la Galería de Arte. 

Start using tildes

File: locales/locales.lua [hunk 9]
```diff
@@ -119,5 +119,59 @@ Locales = {
...
+        ['humanerobbery'] = "Robo de Humane Labs",
+        ['trainrobbery'] = "Robo en el Tren",
+        ['vanrobbery'] = "Robo de Camion Blindado",
+        ['underground'] = "Robo de Bunker",
+        ['drugboatrobbery'] = "Bote Sospechoso",
```
Comment (MonkeyWhisper) [inline locales/locales.lua:172]: Robo de barco de drogas

File: locales/locales.lua [hunk 10]
```diff
@@ -119,5 +119,59 @@ Locales = {
...
+        ['unionrobbery'] = "Robo a Banco Central",
+        ['carboosting'] = "Car Boost en Proceso",
+        ['yachtheist'] = "Alerta en el Yate",
```
Comment (MonkeyWhisper) [inline locales/locales.lua:175]: Assalto al Yate or Atraco de Yate

**Review by MonkeyWhisper — COMMENTED** [review]
There's quite a lot of changes to be done to this translation if you want me to PR it. 

You are not using tildes as well. Try to stick to North American Spanish, not European.

**Comment by MonkeyWhisper** [comment]
No response.

**Pull request closed without merging**